dockers/docker-snmp/start.sh

```diff
@@ -16,11 +16,16 @@ mkdir -p /etc/ssw /etc/snmp
 # Parse snmp.yml and insert the data in Config DB
 /usr/bin/snmp_yml_to_configdb.py
 
+DOCKER0_V4=$(ip -4 addr show docker0 | grep -oP '(?<=inet\s)\d+(\.\d+){3}')
```

Comment:
>docker0 [](http://example.com/codeflow?start=29&length=7)

More secure container will bring better isolation between host and container. There is no access to docker0 insider container in future.

You may consider passing env var while `docker create`.